File: DPR.cpp
#include <iostream>
#include <string>

using namespace std;

class DPR
{
    private:
        string id;
        string name;
        string bidang;
        string partai;

    public:
        DPR()
        {
            this->id = "";
            this->name = "";
            this->bidang = "";
            this->partai = "";
        }

        DPR(string id, string name, string bidang, string partai)
        {
            this->id = id;
            this->name = name;
            this->bidang = bidang;
            this->partai = partai;
        }

        // id
        string get_id()
        {
            return this->id;
        }
        
        void set_id(string id)
        {
            this->id = id;
        }
        
        // nama
        string get_name()
        {
            return this->name;
        }
        
        void set_name(string name)
        {
            this->name = name;
        }
        
        // bidang
        string get_bidang()
        {
            return this->bidang;
        }
        
        void set_bidang(string bidang)
        {
            this->bidang = bidang;
        }
        
        // partai
        string get_partai()
        {
            return this->partai;
        }
        
        void set_partai(string partai)
        {
            this->partai = partai;
        }
        
        
        ~DPR()
        {
            
        }
};

File: PYTHON/DPR.py
class DPR: # deklarasi class yang digunakan
    # atribut private yang digunakan
    __id = ""
    __name = ""
    __bidang = ""
    __partai = ""
    
    # konstraktor
    def __init__(this, id = "", name = "", bidang = "", partai = ""):
        this.__id = id
        this.__name = name
        this.__bidang = bidang
        this.__partai = partai
        
    # aksesor (getter) dan mutator (setter) untuk mengakses atribut-atribut privat
    
    # getter setter untuk atribut id
    def get_id(this): return this.__id
    def set_id(this, id): this.__id = id
    
    # getter setter untuk atribut name
    def get_name(this): return this.__name
    def set_name(this, name): this.__name = name
    
    # getter setter untuk atribut bidang
    def get_bidang(this): return this.__bidang
    def set_bidang(this, bidang): this.__bidang = bidang
    
    # getter setter untuk atribut partai
    def get_partai(this): return this.__partai
    def set_partai(this, partai): this.__partai = partai
